### ISSUE TITLE
After first time login as user Personal Folder is not correctly shown (100000)

### ISSUE THREAD

**Issue by roru69** [nilsteampassnet/TeamPass#2035]
### Steps to reproduce
1. Did a clean install of Teampass Master (24-11-2017 build)
2. Enabled LDAP (which works perfectly now), enabled Personal Folders. Disabled mainenance Mode.
3. Logged in as a LDAP user.

### Expected behaviour
This Screen:

![teampass02](https://user-images.githubusercontent.com/9691381/33203178-791ecad4-d0ff-11e7-961c-782fc31ad224.PNG)

### Actual behaviour
This screen comes up:

![teampass01](https://user-images.githubusercontent.com/9691381/33202958-b571d4c8-d0fe-11e7-9a7b-473ebe9e7482.PNG)

After a refresh of the webpage, all is normal... (F5)

![teampass02](https://user-images.githubusercontent.com/9691381/33202982-cdc8ce64-d0fe-11e7-8073-d5ed2a737021.PNG)

This happens only when the user is newly created, after the first (F5) refresh, all is well. It is a cosmetic issue, but users could think that something is wrong...

### Server configuration
**Operating system**: Debian Jessie

**Web server:**
Apache
**Database:**
MariaDB
**PHP version:**
7.0.x
**Teampass version:** 
2.1.27.10
**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this message the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**
Firefox
**Operating system:**
Windows 10
#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by nilsteampassnet** [comment]
Thank you for reporting, I will investigate

**Comment by go2this** [comment]
Same here. It shows all Personal Folders at each User

![teampass](https://user-images.githubusercontent.com/34249133/33755157-d462918c-dbef-11e7-9968-ad476b0d7ec5.jpg)

**Comment by sshipway** [comment]
I can confirm that I am seeing this as well.  Only on the first login for an LDAP user that did not previously exist in the database.  All the folders are displayed, and personal folders as numbers.  When you enter a salt key, then they all disappear.

**Comment by electrical** [comment]
I can confirm this bug. We currently see this too.

**Comment by R3m1n0X** [comment]
I can confirm this Bug too.

**Comment by nilsteampassnet** [comment]
@jaialrai, would it be possible to use the development branch to check this.

**Comment by nilsteampassnet** [comment]
I think that a good strategy to get rid of this issue is to force the page reload once the user has been added inside Teampass users table.
Such strategy has a drawback, it requires this new user to enter his credentials twice.

The change to perform is the next.
* open the file sources:/identify.php
* replace 
```
        }
        $proceedIdentification = true;
        $user_initial_creation_through_ldap = true;
    }
```
by
```
        }
        $proceedIdentification = true;
        $user_initial_creation_through_ldap = true;

        // Reload page
        echo '[{"value" : "new_ldap_account_created", "user_admin":"", "initial_url" : "", "error" : "", "pwd_attempts" : "0"}]';
        exit();
    }
```

Can you please make me a feedback on this evolution?

**Comment by sshipway** [comment]
Of course it is not ideal for a new user to have to re-enter their credentials (and this could be confusing for new users) but that is still preferable to the current behaviour.

**Comment by jaialrai** [comment]
have tested 👍         $proceedIdentification = true;
        $user_initial_creation_through_ldap = true;

        // Reload page
        echo '[{"value" : "new_ldap_account_created", "user_admin":"", "initial_url" : "", "error" : "", "pwd_attempts" : "0"}]';
        exit();
    }

It does not work, i still get the big list of personal folders and access to ALL passwords as a new user.

**Comment by ethantq** [comment]
I am having the same issue, but the user also sees all passwords in the database. I am on 2.1.27.10 also. 

I tried your code, but it doesn't work. It looks like it's just reloading after creating the users account for ldap user. We need a reload after the user is fully logged in the first time. I am willing to get on a Zoom session with you to troubleshoot. I am central US timezone.

**Comment by nilsteampassnet** [comment]
I did some changes in last commit that should permit to force relog of newly created LDAP user.

**Comment by ethantq** [comment]
New LDAP user is logged out every time now. I see account is being created message the first time and then am prompted to log in again, but every subsequent time I am logged right back out. Existing LDAP users can still sign in.

**Comment by nilsteampassnet** [comment]
Do you use the `development` branch?

**Comment by ethantq** [comment]
Yes, I downloaded the zip of development, ran the upgrade and then tried logging in.

**Comment by nilsteampassnet** [comment]
Issue was identified.
Fix will come.

**Comment by nilsteampassnet** [comment]
Reopen on need

**Comment by jpch** [comment]
Hi, I've upgraded from 2.1.25 mainly due to this bug, but it still happens. The first time some ldap user logs into teampass and I have the personal folders enabled, the user will see the folders and items (although he cannot see the passwords).
as shown here by go2this:
> Same here. It shows all Personal Folders at each User
> 
> ![teampass](https://user-images.githubusercontent.com/34249133/33755206-0a6b7e7e-dbf0-11e7-867e-6b4857891a63.jpg)


After doing a refresh on the page, the items no longer appear.
I've tried the suggestion of adding code to identify.php, but the problem remains.
Thanks.
